dbt/include/bigquery/macros/materializations/materialized_view.sql

```diff
@@ -0,0 +1,71 @@
+{% macro bigquery__get_create_materialized_view_as_sql(relation, sql) %}
+    create materialized view if not exists {{ relation }} as {{ sql }}
+{% endmacro %}
+
+{% macro bigquery__drop_materialized_view_sql(relation) %}
+    drop materialized view if exists {{ relation }}
+{% endmacro %}
+
+{% macro bigquery__get_replace_materialized_view_as_sql(
+    relation,
+    sql,
+    existing_relation,
+    backup_relation,
```

Comment:
`backup_relation` and `intermediate_relation` don't appear to be used anywhere else in `dbt-bigquery` curious if we will have to make default versions of most the chunk parts of matierliazation we define in `adap-2` to be able to handle the changes `base` adapters will have as they don't deal with a lot of pre-built SQL features nateively

Reply:
can we just add `temp_relation` to the input fields as needed? do we also need to do that to dbt-core default versions?